tools/nvmecmd.py

```diff
@@ -0,0 +1,237 @@
+#!/usr/bin/env python
+# SPDX-License-Identifier: Apache-2.0
+#
+# nvmecmd NVMe command observability tool.
+#
+# Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License")
+#
+# 06-Nov-2023   Daniel Gomez   Created this.
+from __future__ import (
+    absolute_import, division, unicode_literals, print_function
+)
+from bcc import BPF
+import argparse
+import logging
+import time
+
+examples = """examples:
+  nvmecmd                          # Observe all NVMe commands
+  nvmecmd --disk nvme9n1           # Observe all commands on 9th NVMe node
+  nvmecmd --ops 0                  # Observe read commands on all NVMe
```

Comment:
'nvmecmd --ops read' is better than 'nvmecmd --ops 0' as people need to go back to check what '0' means? The same for '--ops 1'.